### DISCUSSION TITLE
Is it possible to have linear and logarithmic axis in one line chart

### DISCUSSION THREAD

**Discussion by Alex08151** [Live-Charts/LiveCharts2#227] [Q&A] [answered]
We want to show a graph of temperature and a pressure in one chart. the pressure has a huge range, so we must show it in log scale. Is that possible?
thanks, Alex

**Comment by beto-rodriguez** [comment, marked as answer]
Hi, yes it is possible.

There is no a sample built yet but you can try to create a chart that uses multiple axes:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/ViewModelsSamples/Axes/Multiple/ViewModel.cs

And in any of those axes use the logarithmic scale:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/ViewModelsSamples/Axes/Logaritmic/ViewModel.cs

**Reply by Alex08151** [reply]
That works great! thank you :-)
